readme.txt

```diff
@@ -0,0 +1,41 @@
+=== Plugin Name ===
```

Comment:
Change to AspireUpdate.  This was fixed in #61

Reply:
fixed in this commit https://github.com/aspirepress/AspireUpdate/pull/65/commits/a72c225bedc3bd4ea3e51eb045e772aa9a098b82